examples/rest_api/mlflow_tracking.py

```diff
@@ -0,0 +1,71 @@
+"""
+This simple example shows how we should use MLFlow Rest API for creating new runs and log parameters/metrics.
+For more details see: https://www.mlflow.org/docs/latest/rest-api.html 
+"""
+
+
+import os
+import time
+import requests
+
+#change this base URL so that it points to the host where your MLFlow server is running
```

Comment:
Follow the style of the rest of the code for comments: `#` should have a space after it, and the first word should be capitalized for line and doc comments.